src/client/app/containers/BarChartContainer.ts

```diff
@@ -23,18 +21,26 @@ function mapStateToProps(state: State) {
 			const readingsData = byMeterID[timeInterval.toString()][barDuration.toISOString()];
 			if (readingsData !== undefined && readingsData.readings !== undefined && !readingsData.isFetching) {
```

Comment:
Ths file does [<Button key={1} whereas LineChartContainers.ts has a separate if with a thrown Error. Would be good to be consistent unless there is a reason not to be.

Reply:
Code modified to do this.